docs/static/releasenotes.asciidoc

```diff
@@ -11,6 +11,16 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-0-1]]
```

Comment:
Sorry I didn't catch this before, but you need to add a link for the 6.0.1 relnotes at the top of the file (see line 6).

Reply:
Great catch, thanks @dedemorton !